snuba/datasets/storages/metrics.py

```diff
@@ -143,6 +143,23 @@ def ignore_policy_closure() -> DeadLetterQueuePolicy:
     write_format=WriteFormat.VALUES,
 )
 
+org_counters_storage = ReadableTableStorage(
+    storage_key=StorageKey.METRICS_COUNTERS,
+    storage_set_key=StorageSetKey.METRICS,
+    schema=TableSchema(
+        local_table_name="metrics_counters_v2_local",
+        dist_table_name="metrics_counters_v2_dist",
+        storage_set_key=StorageSetKey.METRICS,
+        columns=ColumnSet(
+            [
+                *aggregated_columns,
+                Column("value", AggregateFunction("sum", [Float(64)])),
```

Comment:
Can I drop this?

Reply:
I would see what happens in tests if you do drop it -- I can't recall if we use the schema columns to do validation or create the return object

Reply:
Tests seem to run fine. I'm not sure whether I should or not, but was mostly trying to limit how much people can do with this entity/storage, since we don't want people making overly complex queries without being restricted to project/org

Reply:
Sounds good, let's only grant access as we need it.

Following up on that, the mappers don't do anything for you if you don't want to surface the sum column.